### PR TITLE
Fix minor formatting and typos in assign-pod-node

### PR DESCRIPTION
**Pull request by glikson**
- few typo's
- slightly unified references to other docs

### CODE REVIEW

File: docs/concepts/configuration/assign-pod-node.md [hunk 1]
```diff
@@ -315,20 +316,15 @@ web-server-1287567482-6f7v5    1/1       Running   0          7m        10.192.4
 web-server-1287567482-s330j    1/1       Running   0          7m        10.192.3.2   kube-node-2
 ```
 
-Best practice is to configure these highly available stateful workloads such as redis with AntiAffinity rules for more guaranteed spreading.
```
Comment (heckj) [inline docs/concepts/configuration/assign-pod-node.md]: did you mean to remove this content?

**Comment by heckj** [comment]
@glikson question about one of the changes to verify it's intended, otherwise lookin' good.. will wait for an lgtm until I hear back

/approve
/assign

File: docs/concepts/configuration/assign-pod-node.md (hunk 1 above)
Comment (glikson) [reply docs/concepts/configuration/assign-pod-node.md]: I did. This section is about 'always co-locate', so it made more sense to me to emphasize the anti-affinity part of this example in the beginning of next section instead.

**Comment by heckj** [comment]
erp, looks like it needs a quick rebase @glikson and you're good to go

**Pull request merged**